### PR TITLE
Feature/duplicate container

### LINKED ISSUE #39: [Feature] Duplicate existing container
It would be useful to have the ability to duplicate an already running and set-up container
instead of relying on either creating from scratch or using `podman/docker` manually.

adding a `--duplicate` flag to `distrobox-create` could be useful.

### PR DESCRIPTION
**Pull request by 89luca89**
Fix #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,6 +211,7 @@ Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
 	--name/-n:		name for the distrobox		default: fedora-toolbox-35
+	--duplicate/-d:		name of the distrobox container to use as base for a new container
```
Comment (alegrey91) [inline README.md:214]: Should be more appropriate to add a deep description in README file about the new feature.

File: distrobox-create [hunk 2]
```diff
@@ -41,6 +42,7 @@ Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
```
Comment (alegrey91) [inline distrobox-create:43]: Remember to remove "default" due to updated line 11.

File: distrobox-create [hunk 3]
```diff
@@ -41,6 +42,7 @@ Options:
...
 	--name/-n:		name for the distrobox		default: fedora-toolbox-35
+	--duplicate/-d:		name of the distrobox container to use as base for a new container
```
Comment (alegrey91) [inline distrobox-create:45]: I'd prefer the name `clone`. What do you think?

**Review by alegrey91 — COMMENTED** [review]
Implementation seems good. I add just some advices.

File: distrobox-create (hunk 2 above)
Comment (89luca89) [reply distrobox-create:43]: Default is now set at line 113 if no duplicate is set

File: distrobox-create (hunk 3 above)
Comment (89luca89) [reply distrobox-create:45]: mmh yea probably it's a bit more pertinent as a name

File: README.md (hunk 1 above)
Comment (89luca89) [reply README.md:214]: Sure i'll add to this later in the file description :+1:

File: README.md [hunk 4]
```diff
@@ -206,12 +206,15 @@ graphical apps (X11/Wayland), and audio.
...
 	distrobox-create --image registry.fedoraproject.org/fedora-toolbox:35 --name fedora-toolbox-35
+	distrobox-create --clone fedora-toolbox-35 --name fedora-toolbox-35-copy
 
 Options:
 
 	--image/-i:		image to use for the container	default: registry.fedoraproject.org/fedora-toolbox:35
 	--name/-n:		name for the distrobox		default: fedora-toolbox-35
-	--duplicate/-d:		name of the distrobox container to use as base for a new container
+	--clone/-c:		name of the distrobox container to use as base for a new container
+				this will be useful to either rename an existing distrobox or have ultiple copies
```
Comment (alegrey91) [inline README.md:216]: You miss "m" in "multiple"

**Review by alegrey91 — COMMENTED** [review]
Fix typo and then LGTM

File: README.md (hunk 4 above)
Comment (89luca89) [reply README.md:216]: aaaaah my typos

**Pull request merged**